tests/data/custom-annotations/mergeable/annotations-ingress.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: custom-annotations-ingress-master
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.org/mergeable-ingress-type: "master"
+    custom.nginx.org/feature-a: "on"
+    custom.nginx.org/feature-b: "512"
+    custom.nginx.org/test-split-feature: "  master.ip--master"
```

Comment:
when we test the split, could we use more common delimiters like `","` instead of `"--"`? otherwise, it will look a little bit more confusing